tests/defer/models.py

```diff
@@ -19,6 +19,14 @@ def __str__(self):
         return self.name
 
 
+class PrimaryOneToOne(models.Model):
```

Comment:
It doesn’t work to change the existing `Primary` model to use a one-to-one field?

Reply:
Changing Primary.related to a `OneToOneField` doesn't work unfortunately, because there are other tests that add more than one Primary for the same Secondary so I'm getting errors. I could instead add a new one-to-one field to Primary that points to Secondary again, do you think that's better?

Reply:
Nah, simpler to have a separate model then.